mne/time_frequency/psd.py

```diff
@@ -112,7 +106,207 @@ def _check_nfft(n, n_fft, n_overlap):
     return n_fft, n_overlap
 
 
+def _check_psd_data(inst, tmin, tmax, picks, proj):
+    """Helper to do checks on PSD data / pull arrays from inst"""
+    from ..epochs import _BaseEpochs
+    from ..io.base import _BaseRaw
+    from ..evoked import Evoked
```

Comment:
whenever I do this I get import errors...any idea how to avoid this w/o nesting the imports?